src/plugins/commander/commands/rc.js

```diff
@@ -0,0 +1,23 @@
+const Command = require('../structs/Command.js');
+
+class RCCommand extends Command {
+    constructor(bot) {
+        super(bot);
+        this.aliases = ['rc', 'rssbot', 'rctodiscord'];
+
+        this.shortdesc = `Information on setting up Recent Changes-to-Discord.`;
+        this.desc = `Tells you how to successfully set up a Recent Changes-to-Discord system.`;
+        this.usages = [
+            '!rc'
+        ];
+    }
+
+    call(message) {
+        message.channel.send(`
+The easiest way to set up a Recent Changes-to-Discord system is by inviting and configuring an RSS bot for your server. A popular one is <https://discordrss.xyz/>.
+When the bot prompts you for the RSS feed/url, enter ${bot.fmt.code('https://<yourwiki>.fandom.com/wiki/Special:Recentchanges?feed=rss')} in. Note that, unlike <#246076868560814080>, the bot will have a small delay, usually around 10 minutes.
```

Comment:
This line will throw an error because `bot` is undefined in this scope, you should do `this.bot`

Also, would be great if you could supply a custom wiki as the command's argument, something like this:
```js
async call(message, content) {
    const link = content
        ? `<https://${await this.bot.fandomizer.url(content)}.fandom.com/wiki/Special:RecentChanges?feed=rss>`
        : this.bot.fmt.code('https://<yourwiki>.fandom.com/wiki/Special:RecentChanges?feed=rss');

    message.channel.send(`blah blah ${link} blah`);
```
Keep in mind that this solution also relies on the fandomizer plugin being added to the list of dependencies, so make sure to update that too (and hey, what do you know, parse.js has them both! what a nice coincidence)